frontend/src/modules/individualDetails/reducers.ts

```diff
@@ -49,7 +49,7 @@ const individualDetailsReducer = (state: WritableDraft<IndividualDetailsById>, a
             const { individualID } = action.meta
             if (!state[individualID]) {
                 state[individualID] = {
-                    individual: undefined,
+                    individual: preprocess(undefined),
```

Comment:
Better not use preprocess here, because it sets the `loaded` property to `true`, but the individual hasn't been received from the backend yet, so it's not loaded. What you want to is an object like this: 
```
{
  id: <individual id goes here>,
  isFetching: true,
  isLoaded: false
}
```